server/bench-wrk/Readme.md

```diff
@@ -0,0 +1,70 @@
+## Benchmarking Hasura GraphQL Engine ##
+
+The script `hge_wrk_bench.py` helps in benchmarking the given version of Hasura GraphQL Engine using a set of GraphQL queries. The results are stored (into the *results GraphQL engine*) along with details like the version of GraphQL engine against which the benchmark is run, the version of Postgres database etc. The stored results can help in comparing benchmarks of different versions of GraphQL engine.
+
+### Setup ###
+
+The setup includes two Postgres databases with [sportsdb](https://www.thesportsdb.com/) schema and data, and two GraphQL engines running on the Postgres databases. Then one of the GraphQL engines is added as a remote schema to another GraphQL engine.
+
+The data will be same in both the databases. The tables reside in different database schema in-order to avoid GraphQL schema conflicts.
+
+The methods in script `sportsdb_setup.py` helps in setting up the databases, starting the Hasura GraphQL engines, and setting up relationships. This script can either take urls of already running Postgres databases as input, or it can start the databases as Docker instances. The GraphQL engines can be run either with `stack exec` or as Docker containers.
+
+### Run benchmark ###
+- Install dependencies for the Python script in a virtual environment (requires Python>=3.6).
```

Comment:
Using `pyenv` to handle python versions worked well for me (thanks for the tip!)